oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/meter/function/AvgHistogramFunction.java

```diff
@@ -157,6 +165,7 @@ public void deserialize(final RemoteData remoteData) {
 
         remoteBuilder.addDataObjectStrings(count.toStorageData());
         remoteBuilder.addDataObjectStrings(summation.toStorageData());
+        remoteBuilder.addDataObjectStrings(dataset.toStorageData());
```

Comment:
Do you forget to serialize this field?

Reply:
yep. With the unit test's help, I found this issue.